etc/answers-core.api.md

```diff
@@ -287,6 +285,21 @@ export interface NearFilterValue {
     radius: number;
 }
 
+// @public
+export interface NumberRangeLimit {
+    matcher: NumberRangeMatcher;
+    value: number;
+}
+
+// @public
+export type NumberRangeMatcher = Matcher.GreaterThan | Matcher.GreaterThanOrEqualTo | Matcher.LessThan | Matcher.LessThanOrEqualTo;
+
+// @public
+export interface NumberRangeValue {
+    end: NumberRangeLimit;
+    start: NumberRangeLimit;

```

Comment:
What do you think of making these optional? The spec says that if an 'end' or a 'start' isn't specified, the range is unbounded

Reply:
From the first example facet in spec, it looks like it's always bounded in the backend. However, after discussing with product, the backend will handle the case when one of the limit is not specified. I will update this